core/src/main/scala/org/dbpedia/extraction/config/dataparser/DateTimeParserConfig.scala

```diff
@@ -6,6 +6,8 @@ object DateTimeParserConfig
     val monthsMap = Map(
         // For "ar" configuration, right-to-left rendering may seem like a bug, but it's not.
         // Don't change this unless you know how it is done.
+        "am" -> Map("january"->1,"february"->2,"march"->3,"april"->4,"may"->5,"june"->6,"july"->7,"august"->8,"september"->9,"october"->10,"november"->11,"december"->12, 
```

Comment:
Not sure if doing uppercase is correct since all other language mappings are lowercase, 
IIRC we lowercase our input before processing and this could change could skip the matching